stdlib/array.gr

```diff
@@ -816,6 +902,9 @@ provide let zipWith = (fn, array1: Array<a>, array2: Array<b>) => {
  *
  * @param array: The array of tuples to split
  * @returns An array containing all elements from the first tuple element, and an array containing all elements from the second tuple element
+ * 
+ * @example
+ * Array.unzip([> (1, 2), (3, 4), (5, 6)]) // ([> 1, 3, 5], [> 2, 4, 6])
```

Comment:
Same As Above:

```suggestion
 * Array.unzip([> (1, 2), (3, 4), (5, 6)]) == ([> 1, 3, 5], [> 2, 4, 6])
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....